util/network/src/discovery.rs

```diff
@@ -217,7 +217,7 @@ impl Discovery {
 				self.send_packet(PACKET_FIND_NODE, &r.endpoint.udp_address(), &rlp);
 				self.discovery_nodes.insert(r.id.clone());
 				tried_count += 1;
-				debug!(target: "discovery", ">>RLPX_FIND_NODE to {:?}", &r.endpoint);
+				debug!(target: "discovery", ">>FINDNODE/v4 to {:?}", &r.endpoint);
```

Comment:
I prefer to actually call this `RLPX_FINDNODE` across both parity and geth, since it's more descriptive and consistent with `DEVP2P_` and `ETH_` messages. @SimonSK what do you think?

Reply:
yea. now i look at it, RLPX_ seems more consistent than /v4. so rollback sid's change and i change geth's logging to follow this pattern?

Reply:
Yup - sounds good to me.